code/game/objects/items.dm

```diff
@@ -594,7 +594,8 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 		var/obj/item/organ/internal/eyes/eyes = H.get_int_organ(/obj/item/organ/internal/eyes)
 		if(!eyes) // should still get stabbed in the head
 			var/obj/item/organ/external/head/head = H.bodyparts_by_name["head"]
-			head.receive_damage(rand(10,14), 1)
+			if(head)
```

Comment:
Lots of potentially unsafe code here. I can't see any safeties in `bodyparts_by_name` so we might have to do it ourselves

Reply:
Oh god, you're right. I'll have to take a runthrough to see where that crops up.

Reply:
For the record, I think the checks look alright, and I've gone through and added clarity to some more.